cmd/version.go

```diff
@@ -3,28 +3,68 @@ package cmd
 import (
 	"fmt"
 
+	"github.com/exercism/cli/cli"
 	"github.com/spf13/cobra"
 )
 
 // Version is the version of the current build.
 // It follows semantic versioning.
 const Version = "3.0.0-alpha.1"
 
+// checkLatest flag for version command.
+var checkLatest bool
+
 // versionCmd outputs the version of the CLI.
 var versionCmd = &cobra.Command{
 	Use:     "version",
 	Aliases: []string{"v"},
 	Short:   "Version outputs the version of CLI.",
 	Long: `Version outputs the version of the exercism binary that is in use.
 
-To see if there is a more recent version available, call the command with the
+To check for the latest available version, call the command with the
 --latest flag.
 	`,
+
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println(Version)
+		fmt.Println(currentVersion())
+
+		if checkLatest {
+			c := cli.New(Version)
+			l, err := checkForUpdate(c)
+			BailOnError(err)
+
+			fmt.Println(l)
+		}
+
+		return
 	},
 }
 
+// currentVersion returns a formatted version string for the Exercism CLI.
+func currentVersion() string {
+	return fmt.Sprintf("exercism version %s", Version)
+}
+
+// checkForUpdate returns a string indicating if the current running version of the Exercism CLI is at the latest release.
+// For CLI versions found to be out of date a string with upgrade instructions is included in the return.
+func checkForUpdate(c *cli.CLI) (string, error) {
+
+	ok, err := c.IsUpToDate()
+	if err != nil {
+		return "", err
+	}
+
+	if ok {
+		return "Your CLI version is up to date.", nil
+	}
+
+	// Anything but ok is out of date.
+	o := fmt.Sprintf("A new CLI version is available. Run `exercism upgrade` to update to %s", c.LatestRelease.Version())
+	return o, nil
```

Comment:
I don't know what `o` means. For a `string` the single-letter variable name would typically be `s`.

Here I would probably use `msg`, since that's what it feels like. I don't feel strongly about `s` vs `msg`, but `o` makes me do a double-take and try to figure out what it stands for.